harness/chaos/faults/isolate_client_from_cluster.py

```diff
@@ -0,0 +1,37 @@
+from time import sleep
+from sh import ssh
+import logging
+from chaos.faults.types import FaultType
+
+logger = logging.getLogger("chaos")
+
+class IsolateClientFromCluster:
```

Comment:
We do more than just isolate a client from a cluster. Let's call it something like TriggerKIP360